Content: MQTT Announcement (#2721)
FlowFuse/website: src/blog/2024/10/announcement-mqtt-broker.md

---

> title: "Announcement: MQTT Service Now Available on FlowFuse"

Reviewer: ```suggestion
title: "MQTT Service Now Available on FlowFuse"
```

subtitle: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.
description: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.
date: 2024-10-30
authors: ["joe-pavitt"]
image: /blog/2024/10/images/dashboard-1-18-0-blog-tile.png
tags:
   - posts
   - news
   - flowfuse
   - node-red
---

> You may have seen in our [recent product update](/blog/2024/10/flowfuse-release-2-10) that we have release our very own MQTT service, built-in and ready to use alongside your Node-RED applications. We are always engaging with users and prospective customers and this has consistently been a request, and so we are delighted to announce that this is now live on FlowFuse Cloud for our Team and Enterprise teams.

Reviewer 2: "we have release our very own MQTT service"

Apart from "release" should be "released", whould this not be that we have "we have added" instead?

Author: Sensible, feel free to make the change when you're adding in the other paragraph

Reviewer 2: ```suggestion
You may have seen in our [recent product update](/blog/2024/10/flowfuse-release-2-10) that we have added our very own MQTT service, built-in and ready to use alongside your Node-RED applications. We are always engaging with users and prospective customers and this has been a highly requested feature, and so we are delighted to announce that this is now live on FlowFuse Cloud for our Team and Enterprise teams.
```

This is my take Joe.

<!--more-->

The MQTT Service is available now on FlowFuse Cloud. FlowFuse permits you to setup your own secure clients to begin publishing and subscribing to your own topics.

You can now use FlowFuse to manage your own MQTT Clients alongside your Node-RED instances, making it easier to build full-stack, event-driven applications within FlowFuse.

> 

Reviewer: Can we add a little bit more information here on the role MQTT plays in industrial applications/deployments. A use case or two will be helpful to understand more on why we added it to the platform.

Reviewer 2: @joepavitt I will review the doc structure and see what fits the narrative.

Reviewer 2: ```suggestion

## Use Cases

* Connect your MQTT enabled PLCs (like Omron N Series, Siemens S7, etc) to your Node-RED instance to open up such possibilities like:
    * Data collection and analysis like Predictive Maintenance, OEE, Condition Based Monitoring. 
    * Triggering actions like sending Emails or alerting your engineers about an event.
    * Realtime production monitoring of your facility.
* Make use of modern IIoT devices on legacy systems by bridging the gap with MQTT.
* Connect disparate systems together where Node-RED and FlowFuse act as the central hub for data processing and routing. This gives you the advantage of transforming data on the fly, applying extra contextual data from other systems, apply routing rules, and much more.

```

@joepavitt @Hasmin-AC - for your consideration

Author: Reads good to me - @Hasmin-AC?

---
title: "MQTT Service Now Available on FlowFuse"
subtitle: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.
description: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.

> date: 2024-10-30

Reviewer 3: ```suggestion
date: 2024-10-31
```

authors: ["joe-pavitt"]
image: /blog/2024/10/images/ff-mqtt.png
tags:
   - posts
   - news
   - flowfuse
   - node-red
---

You may have seen in our [recent product update](/blog/2024/10/flowfuse-release-2-10) that we have added our very own MQTT service, built-in and ready to use alongside your Node-RED applications. We are always engaging with users and prospective customers and this has been a highly requested feature, and so we are delighted to announce that this is now live on FlowFuse Cloud for our Team and Enterprise teams.

<!--more-->

> 

Reviewer: MQTT is the lightweight messaging protocol ideal for efficient communication in industrial settings. Now with FlowFuse you can collect, transport, visualize and interact with your industrial data, all in one platform.

> The MQTT Service is available now on FlowFuse Cloud. FlowFuse permits you to setup your own secure clients to begin publishing and subscribing to your own topics.

Reviewer: ```suggestion
The MQTT Service is available now on [FlowFuse Cloud](https://flowfuse.com/product/cloud/). FlowFuse permits you to setup your own secure clients to begin publishing and subscribing to your own topics.
```

You can now use FlowFuse to manage your own MQTT Clients alongside your Node-RED instances, making it easier to build full-stack, event-driven applications within FlowFuse.

## Use Cases

> FlowFuse MQTT Service facilitates real-time communication, an essential element for optimizing industrial processes. This service allows for the monitoring of production lines and the precise control of processes, leading to enhanced efficiency. Its seamless integration with existing industrial systems simplifies data collection, which can then be analyzed to improve decision-making and boost productivity. Here are some typical uses cases:

Reviewer 2: Not sure why this section was added. TBH, IMHO, it feels dangerously close to bordering on "waffle".

Yes, MQTT (like almost any technology) can be used to "improve production" but it is not a magic bullet.

Reviewer: It's to give the use cases some context. this is meant to be promotional post ;)

Reviewer 2: Hi Hasmin, sure, i get it, but when I read that section (as a tech person who has done all of the use-cases in real-world scenarios using tech like Direct DB, OPC, SCADA etc) it just had a "smell" - one i suspect others would detect too.

For example: Just because a facility gets MQTT capability, you do not suddenly end up with "the monitoring of production lines" or "precise control of processes". Nether is it seamless (MQTT is not a discoverable protocol).

I think a little less fluff will come across better.

Just my 2C

Reviewer 2: ```suggestion
FlowFuse MQTT Service simplifies access to real-time data, an important element in optimizing industrial processes. With the power of FlowFuse, Node-RED and the MQTT service, your integrations are now even easier. Here are some typical uses cases:
```

1. Changed the focus from  "real-time communication" to "real-time data" as the value is in the data not the act of communicating.
1. I dont spell out "simplifies data collection" because in reality, you still need somewhere to put it (we dont currently offer a DB)
1. I have removed "precise control of processes" (MQTT is rarely used for process control without safety controllers and such) - I felt it was better to leave that out
1. I have simplified the overall statement because much of the improvements are implicit within the use casees (e.g. OEE stands for Overall Equipment Effectiveness, Predictive Maintenance leads to savings, condition Based monitoring improves MBD time, etc)

WDYT? Happy to iterate.
